src/components/sdfg_viewer.ts

```diff
@@ -497,14 +482,42 @@ export class SdfgViewerProvider
             });
 
             // Handle received messages from the webview.
-            webviewPanel.webview.onDidReceiveMessage(message => {
-                ComponentMessageHandler.getInstance().handleMessage(
-                    message,
-                    webviewPanel.webview
-                );
-            });
+            editor.register(this.disableMinimap, this);
```

Comment:
I thought these get auto-registered in the decorator? weird

Reply:
They are now, I forgot to adopt the change..